pkg/minikube/constants/constants.go

```diff
@@ -55,10 +59,24 @@ func MakeMiniPath(fileName ...string) string {
 	return filepath.Join(args...)
 }
 
+// GetProfileHomeDir determines the Minishift home directory for the profile.
+func GetProfileHomeDir() string {
+	if ProfileName != DefaultProfileName {
+		homeEnv, ok := os.LookupEnv(MiniShiftHomeEnv)
+		if ok {
+			return filepath.Join(homeEnv, "profiles", ProfileName)
+		} else {
+			return filepath.Join(util.HomeDir(), ".minishift", "profiles", ProfileName)
+		}
+	} else {
```

Comment:
Same here. We can write `return GetMinishiftHomeDir()` after if. No else required.

Reply:
I wasn't going to nitpick about this, but I agree. Nesting and using `else` when not necessary make the code heard to reason about.

https://blog.mozilla.org/nnethercote/2009/08/31/no-else-after-return-considered-harmful/